opencti-platform/opencti-graphql/src/resolvers/stixDomainObject.js

```diff
@@ -96,7 +95,7 @@ const stixDomainObjectResolvers = {
           () => pubSubAsyncIterator([bus.EDIT_TOPIC, bus.CONTEXT_TOPIC]),
           (payload) => {
             if (!payload) return false; // When disconnect, an empty payload is dispatched.
-            return payload.user.id !== context.user.id && payload.instance.id === id;
+            return payload.instance.id === id;
```

Comment:
I think this could be an issue, because we don't want to subscribe to all edit events for current user (to be confirmed @richard-julien)